modules/invamiaBidAdapter.js

```diff
@@ -0,0 +1,78 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'invamia';
+const ENDPOINT_URL = 'https://ad.invamia.com/delivery/impress';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bidRequest The bid request params to validate.
+   * @return boolean True if this is a valid bid request, and false otherwise.
+   */
+  isBidRequestValid: function(bidRequest) {
+    return !!bidRequest.params.zoneId;
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {Array<BidRequest>} validBidRequests an array of bid requests
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(validBidRequests) {
+    const bidRequest = validBidRequests[0];
+    const {bidId, mediaTypes: {banner: {sizes: [[width, height]]}}} = bidRequest;
```

Comment:
This line will only take first sizes from `bidRequest.mediaTypes.banner.sizes`, it will ignore other sizes.  
So, if you define adUnit like this:

```
const adUnits = [{
    code: 'div-gpt-ad-1460505748561-0',
    mediaTypes: {
        banner: {
            sizes: [[300, 250], [300, 600]],
        },
    }
}
```

Size 300x600 is ignored.